pages/register.tsx

```diff
@@ -0,0 +1,47 @@
+import { ReactElement } from "react";
+import Head from "next/head";
+
+export default function Register(): ReactElement {
+  return (
+    <>
+      <Head>
+        <title>CUSEC 2021 Registration</title>
+        <meta
+          name="description"
+          content="Register for Canada's favourite conference! Join us January 9-10, 2021 for a weekend of amazing speakers, workshops, and more!"
+        />
+      </Head>
+
+      <iframe
+        id="typeform-full"
+        width="100%"
+        height="100%"
+        frameBorder="0"
+        allow="camera; microphone; autoplay; encrypted-media;"
+        src="https://eli030.typeform.com/to/FFoe9hQ9"
+      ></iframe>
```

Comment:
Kind of surprised ESLint didn't say anything about using a self-closing tag here (and below script) 😂

Reply:
Yeah 😂 I pretty much pasted the exact embed code typeform provided and it didn't complain